add_up.rb

```diff
@@ -0,0 +1,21 @@
+# define a method name add_up and it returns the sum of numbers
+def add_up(number1, number2)
```

Comment:
The assignment says to implement the function `add_up(i)`, which only accepts one parameter, but you implemented something else.

A couple of examples of what that function should return:
* `add_up(2)` should return `3` because 1 + 2 = **3**
* `add_up(4)` should return `10` because 1 + 2 + 3 + 4 = **10**